scripts/e2e-host-controller.sh

```
@@ -0,0 +1,509 @@
+  export DENSE_MEM_IMAGE="$image"
+  export DENSE_MEM_CI_CONTRACT="$CONTRACT_VERSION"
+  export DENSE_MEM_CI_REPOSITORY="$REPOSITORY"
+  export DENSE_MEM_CI_RUN_ID="$run_id"
+  export DENSE_MEM_CI_RUN_ATTEMPT="$attempt"
+  export DENSE_MEM_CI_PHASE="$phase"
+  export DENSE_MEM_CI_SCENARIO="$scenario"
+  export DENSE_MEM_CI_IMAGE_DIGEST="$digest"
+  export DENSE_MEM_CI_CREATED_AT="$created_at"
+  export DENSE_MEM_CI_PROMETHEUS_FILE="$PROMETHEUS_FILE"
+  export DENSE_MEM_CI_TELEMETRY_TOKEN_FILE="$TELEMETRY_TOKEN_FILE"
+  export DENSE_MEM_CI_TELEMETRY_SCRAPE_TOKEN="$(cat "$TELEMETRY_TOKEN_FILE")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)